src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -889,9 +889,12 @@ private void WorkspaceAdded(WorkspaceModel item)
             var newVm = new WorkspaceViewModel(item, this);
             if (item is HomeWorkspaceModel)
             {
+                (item as HomeWorkspaceModel).DynamicRunEnabled = DynamicRunEnabled;
```

Comment:
I am not sure if this is a no-op, `DynamicRunEnabled` (the right hand side) is as follow:

```csharp
public bool DynamicRunEnabled
{
    get
    {
        return HomeSpace.DynamicRunEnabled;
    }
}

public HomeWorkspaceModel HomeSpace
{
    get
    {
        return model.Workspaces.OfType<HomeWorkspaceModel>().FirstOrDefault();
    }
}
```

When we get to `WorkspaceAdded(WorkspaceModel item)` method, the `model.Workspaces` would have been updated with the incoming `item` here. In another word, `item` here is the same as `HomeSpace`, so the above assignment may not do anything. Am I right?